### ISSUE TITLE
Multiplayer AI doesn't respond to attacks on serfs

### ISSUE THREAD

**Issue by Thiez** [Kromster80/kam_remake#107]
Yesterday I was playing a multiplayer game with a friend against a single AI enemy. At one point in the game I managed to sneak into the AIs base with a single milita unit. Figuring I'd only have a short time to enjoy this advantage, I parked him in front of the entrance of the AI's castle (the one that recruits military units, not a magazine). There he stood for several minutes, killing what must have been dozens of serfs that were trying to bring valuable metal weapons and armor to the castle. At some point I felt like I was exploiting a bug, but I couldn't even move the unit away because there was a steady stream of serfs that would stand into melee range, and you can't control melee units when they are in combat.

The AI had several pikemen, bowmen, and crossbowmen available, and perhaps some swordsmen as well. None of them moved in to kill my single milita.

Version r6720.

**Comment by Kromster80** [comment]
This is likely due to the fact that your offense was outside of AI defense lines retaliation radius.

Solution needs some planning ..

**Comment by Thiez** [comment]
Sorry for the accidental close! So the AI only defends within a certain area, and it is possible for that area to not include parts of its own base?

**Comment by Kromster80** [comment]
Yes, iirc AI defense groups radiuses are constant and may or may not cover the whole of AI town.

We will need to find a solution for that issue. Thanks for reporting!
